test/integration/targets/nxos_zone_zoneset/tests/common/sanity.yaml

```diff
@@ -0,0 +1,207 @@
+---
+- debug: "msg=\"START nxos_zone_zoneset sanity test with connection={{ ansible_connection }} \""
+- debug: "msg=\"Using vsans {{ vsan1 }}, {{ vsan2 }} for running this sanity test, please make sure these are not used in the setup, these will be deleted after the tests\""
+- always:
+    - name: "Remove vsan config"
+      nxos_vsan:
+        vsan:
+          - id: "{{ vsan1 | int }}"
+            remove: true
+          - id: "{{ vsan2 | int }}"
+            remove: true
+  block:
+    - ignore_errors: true
+      name: "Setup - Remove vsan if configured"
+      nxos_vsan:
+        vsan:
+          - id: "{{ vsan1 | int }}"
+            remove: true
+          - id: "{{ vsan2 | int }}"
+            remove: true
+    - name: "Configure vsan"
+      nxos_vsan:
+        vsan:
+          - id: "{{ vsan1 | int }}"
+          - id: "{{ vsan2 | int }}"
+    - name: "Configure zone and zoneset"
+      nxos_zone_zoneset:
+        zone_zoneset_details:
+          - mode: enhanced
+            vsan: "{{ vsan1 | int }}"
+            zone:
+              - members:
+                  - pwwn: "11:11:11:11:11:11:11:11"
+                  - device_alias: test123
+                  - pwwn: "61:61:62:62:12:12:12:12"
+                    remove: true
+                name: zoneA
+              - members:
+                  - pwwn: "10:11:11:11:11:11:11:11"
+                  - pwwn: "62:62:62:62:21:21:21:21"
+                name: zoneB
+              - name: zoneC
+                remove: true
+            zoneset:
+              - action: activate
+                members:
+                  - name: zoneA
+                  - name: zoneB
+                  - name: zoneC
+                    remove: true
+                name: zsetname1
+              - action: deactivate
+                name: zsetTestExtra
+                remove: true
+          - mode: basic
+            smart_zoning: true
+            vsan: "{{ vsan2 | int }}"
+            zone:
+              - members:
+                  - devtype: both
+                    pwwn: "11:11:11:11:11:11:11:11"
+                  - pwwn: "62:62:62:62:12:12:12:12"
+                  - devtype: both
+                    pwwn: "92:62:62:62:12:12:1a:1a"
+                    remove: true
+                name: zone21A
+              - members:
+                  - pwwn: "10:11:11:11:11:11:11:11"
+                  - pwwn: "62:62:62:62:21:21:21:21"
+                  - device_alias: somedummyname
+                  - device_alias: anydummyname
+                    remove: true
+                name: zone21B
+            zoneset:
+              - action: activate
+                members:
+                  - name: zone21A
+                  - name: zone21B
+                name: zsetname21
+      register: result
+    - assert:
+        that:
+          - "result.changed == true"
+    - name: "Idempotence Check"
+      nxos_zone_zoneset:
```

Comment:
```suggestion
      nxos_zone_zoneset: *zoneset
```

Reply:
This will allow reusing the previous task in the idempotence tests. Valid for all the integration tests in this PR.

Reply:
done